new/PRS13770EntonyosWaldaAbadia.xml

```diff
@@ -55,6 +58,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                 </person>
                 <listRelation>
                     <relation name="saws:hasOwned" active="PRS13770EntonyosWaldaAbadia" passive="BNFabb122"/>
+                    <relation name="saws:isRelatedTo" active="PRS13770EntonyosWaldaAbadia" passive="PRS1071dAbbadi" cert="medium"/>
```

Comment:
saws relations are for texts
possibly `<relation name="syriaca:possibly-identical"></relation>`?
(to me it seems relatively certain)

Reply:
Yes, but theoretically an Ethiopian name like this is possible. It will be certain when I come across his Ethiopic writign and it will be the same. It will happen sooner or later.